Components/Typography/typography.css

```diff
@@ -0,0 +1,117 @@
+.typ-ex-grid {
+  display: grid;
+  grid-template-columns: max-content min-content;
+  grid-template-rows: repeat(auto, 50px);
```

Comment:
If you check in browser, grid-template-rows is not getting applied because it is either auto-fit or auto-fill. Plus you don't need repeat with auto because height is generally unlimited, it is generally used for columns. You can use grid-template-rows: 50px; , will give same effect.

Reply:
oh okay! didnt know auto-fit was a thing will go check it out :)